apps/hashdotdev/src/pages/roadmap/technology-tree-data.tsx

```diff
@@ -576,7 +576,7 @@ export const technologyTreeData: TechnologyTreeNodeData[] = [
     heading: "Multi-Type Entities",
     body: "Allow a single entity to have more than one assigned type",
     parentIds: ["32"],
-    status: "future",
+    status: "in-progress",
```

Comment:
Is this intentional? Type inheritance is already in progress, I don't think multi-type entities should be / is

Reply:
@CiaranMn It was intentional. A form of multi-type entities are inherently supported via type inheritance. I've opened a Slack thread to discuss the separate matter of adding multiple types directly to an entity. Let's continue the conversation there or offline. We can leave this thread unresolved until we've reached a conclusion, however, re: what is most appropriate here.

Reply:
```suggestion
    status: "next-up",
```

Per [discussion in Slack](https://hashintel.slack.com/archives/C03F7V6DU9M/p1692174122792159?thread_ts=1692125255.951789&cid=C03F7V6DU9M) (internal link).